config/application.rb

```diff
@@ -121,6 +121,13 @@ class Application < Rails::Application
     #
     #   https://bugs.ruby-lang.org/issues/14372
     #
+
+    # TODO: Remove this once we move to config.load_defaults 7.0 as this is the default.
+    # Note, rails 7 can read cache format from 6 or 7 so there is no risk if you're running rails 7.
+    # See: https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#new-activesupport-cache-serialization-format
+    warn "Warning: Remove redundant config.active_support.cache_format_version = 7.0 from #{__FILE__}:#{__LINE__ + 1} if using config.load_defaults 7.0" if config.active_support.cache_format_version == 7.0
```

Comment:
note, this is nil if you load defaults earlier than 7.0

Reply:
This warning will only occur when we're on rails 7.0+ where we're using `config.load_defaults 7.0` or newer.  It's a reminder to remove this setting when https://github.com/ManageIQ/manageiq/pull/23176 is merged.